MapboxNavigation/FeedbackViewController.swift

```diff
@@ -305,3 +326,225 @@ extension String {
         return ceil(boundingBox.height)
     }
 }
+
+public class FeedbackSubtypeViewController: FeedbackViewController {
```

Comment:
Worth updating `User Feedback` section in `jazzy.yml` to contain `FeedbackSubtypeViewController` and modifying `CHANGELOG.md`.

Reply:
Done.